Source/WebKit/WebProcess/WebPage/mac/WKAccessibilityWebPageObjectBase.h

```diff
@@ -53,7 +53,7 @@ class AXCoreObject;
 #if ENABLE(ACCESSIBILITY_ISOLATED_TREE)
 - (void)setPosition:(const WebCore::FloatPoint&)point;
 - (void)setSize:(const WebCore::IntSize&)size;
-- (void)setIsolatedTreeRoot:(NakedPtr<WebCore::AXCoreObject>)root;
+- (void)setIsolatedTreeRoot:(RefPtr<WebCore::AXCoreObject>)root;
```

Comment:
This change doesn't look right. Our policy is to ref at call sites, not change the type of parameters.

Reply:
Sounds good, I'll go ahead and ignore this static analyzer result (it flagged WKAccessibilityWebPageObjectBase.mm:145)

Reply:
Yes, that warning doesn't seem right to me.